httpd/handler_test.go

```diff
@@ -146,7 +146,7 @@ func TestHandler_RetentionPolicies(t *testing.T) {
 
 	if status != http.StatusOK {
 		t.Fatalf("unexpected status: %d", status)
-	} else if body != `{"results":[{"rows":[{"columns":["Name"],"values":[["bar"]]}]}]}` {
```

Comment:
Do we really want capitalized column names? It wasn't clear to me. Easy to revert if so.